cpp/src/detail/shuffle_wrappers.cu

```diff
@@ -18,7 +18,7 @@
 #include <cugraph/partition_manager.hpp>
 #include <cugraph/utilities/shuffle_comm.cuh>
 
-#include <raft/random/rng.cuh>
+#include <raft/random/rng.hpp>
```

Comment:
This one can be deleted, this file does not use raft's random number generator.

Reply:
Done